docs/data-sources/vlans.md

```diff
@@ -17,7 +17,7 @@ Provides details about Linode VLANs.
 ```terraform
 resource "linode_instance" "my_instance" {
   label      = "my_instance"
-  image      = "linode/ubuntu18.04"
+  image      = "linode/ubuntu23.10"
```

Comment:
Can we change it to `22.04` for a longer supported time?
You can check Ubuntu lifecycle here: https://ubuntu.com/about/release-cycle

Reply:
Sure, thanks for pointing it out!